omnizart/constants/datasets.py

```diff
@@ -165,7 +165,7 @@ def download(cls, save_path="./"):
         or filter out some files.
         """
         ensure_path_exists(save_path)
-        save_name = cls.__name__.replace("Structure", "") + ".zip"
+        save_name = f"{cls.__name__.replace('Structure', '')}.zip"
```

Comment:
Please revert this change. The original one is more clear than the f-string one.

Reply:
Sure, agreed. Just reverted it.